README.md

```diff
@@ -26,80 +26,68 @@
-
-For `dev` and `prod` targets you can authinicate using `sfdx force:auth:web:login  --setdefaultusername -a your-alias`.
+### Existing Apps
+Before you can run a new app that isn't in your target org, you'll fire need to do a deploy to get the dependent salesforce metdata over there using `npm run deploy-dev`
 
-For new scratch orgs use: `sfdx force:org:create -a your-alias`
+#### Default Target
 
-*Note: if you are trying to connect to a sandbox, make sure to set the instance url with the `-r` arg*
-
-#### changing between orgs
-
-To change your default target, simply run one of the following commands:
-
-```npm
+Sever commands have a default target. Use the following commands to change the defaults to the desired alias listed in `.npmrc`
+```
 npm run make-dev-default
 npm run make-scratch-default
 npm run make-prod-default
 ```
 
 ## DEVELOPMENT
+Once your salesforce dependencies exist in your target org things get a bit more interesting.
 
-### localhost with HMR (hot module reloading)
-
-[See HMR in action](http://i.imgur.com/j9NBbmf.gif)
-
-Hot module reloading means that your updates will injected into your app without having to refresh the page or even lose state (in some cases)
+### Run Locally with HMR (hot module reloading)
+First you can host you're app on localhost and enjoy hot module reloading, and use salesforce data from the current default target. Any updates to your app will immediately show up without a pa ge refresh. Your state is also preserved in most cases.[See HMR in action](http://i.imgur.com/j9NBbmf.gif). This workflow is well suited for playing around with change without needing them to be live in your salesforce environment, i.e. if you're fixing a bug in sandbox while other users are current testing you app in different areas. By developing locally you don't have to worry about a mistake during your development impacting the users testing directly on Salesforce.
 
-1. [Deploy to target](#deployment) (to setup objects)
-1. whitelist localhost CORS on the target ORG: `npm cors-enable`
-1. `npm start`
+1. `npm run deploy-dev` (deploy salesforce dependencies if needed)
+1. `npm run cors-enable` (whitelist localhost CORS on the default target org) DANGER*
+1. `npm start` (start a local webserver with hot reload)
 
-* WARNING: if you do this on production make sure to disable cors when done with `npm cors-disable`! [why?](https://stackoverflow.com/questions/39042799/cors-localhost-as-allowed-origin-in-production)*
+* DANGER: while allowing salesforce to accept request from a localhost server is awesome for hot reloading it has security risks. It's best if you don't do this in a prodcution or org with sensitive data. But if you did, be sure to disable cors when done with `npm run cors-disable` to disable the security hole!!! [why?](https://stackoverflow.com/questions/39042799/cors-localhost-as-allowed-origin-in-production)*
 
-### dev/scratch org with local asset
+### Run Remotely With Local Assets (dev/scratch org only)
```

Comment:
you can actually do this in a production org as well... A usecase would be trying to hotfix a bug you can't repro in sandbox.  Since the app resource is controlled using a url param, end users are not impacted.  (of course you have to enable cors)